src/package/packageConvert.ts

```diff
@@ -164,20 +162,21 @@ export async function createPackageVersionCreateRequest(
   const metadataZipFile = path.join(packageVersBlobDirectory, 'package.zip');
   const packageVersBlobZipFile = path.join(packageVersTmpRoot, 'package-version-info.zip');
 
-  const packageDescriptorJson = {
+  const packageDescriptorJson: PackageDescriptorJson & {
+    [prop: string]: unknown;
+  } = {
     id: packageId,
   };
 
   const settingsGenerator = new SettingsGenerator({ asDirectory: true });
   const definitionFile = context.definitionfile;
-  let definitionFileJson: ScratchOrgInfo;
+  let definitionFileJson: ScratchOrgInfo & { [prop: string]: unknown };
```

Comment:
what made this necessary?  Do you know the additional props are **that are used by this code?**